### PR TITLE
#185106846 feature Users should be able to Login via Google. 

### PR DESCRIPTION
**Pull request by divinecharlotte**
### What does this PR do?

- This PR is for allowing users to log in via Google. 

### How should this be manually tested?

**Installation**
**1. Clone the repository by running** 
```
git clone https://github.com/atlp-rwanda/destructors-ec-fe.git
```
**2. Navigate to the project directory by running **
```
cd destructors-ec-fe
```
```
git checkout ft-goodle-login-185106846
```
**3. Install the necessary dependencies by running **
```
npm install
```
**6. Run react  development server by running**
```
npm run dev
```

# # # Some background context?
go to this route `http://localhost:5173/auth/signup` to check how it works
from that end point you will use the google button 

add this in your .env

```
VITE_BASE_URL = http://localhost:3000/api/v1/users
VITE_GOOGLE_LOGIN_APP_API_URL = /login/google
```
### Questions:

- none

### CODE REVIEW

**Review by istafrica-dennw — CHANGES_REQUESTED** [review]
1. PR named Wrong

File: src/components/GoggleLogin.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Google from '../assets/Google.png'
+function GoogleLogin() {
+  const handleLogin = () => {
+    window.location.href = 'http://localhost:3000/api/v1/users/login/google';
+  };
```
Comment (leandreAlly) [inline src/components/GoggleLogin.jsx:10]: use environment variables instead of hard coding values

**Review by leandreAlly — CHANGES_REQUESTED** [review]

**Comment by divinecharlotte** [comment]
> 1. PR named Wrong
hello @denislohan 
 

- [x] renamed well

**Comment by ba-lambert** [comment]
@divinecharlotte how this pr is tested cz this endpoint you gave us contains nothing

**Review by ba-lambert — COMMENTED** [review]
how this pr is tested cz this endpoint you gave us contains nothing

**Comment by divinecharlotte** [comment]
> how this pr is tested cz this endpoint you gave us contains nothing

@ba-lambert  check pr description updated

**Review by ba-lambert — APPROVED** [review]

**Review by JENNY-257 — APPROVED** [review]

**Review by katros1 — APPROVED** [review]

**Pull request merged**